content/pages/docs-as-code.html

```diff
@@ -0,0 +1,266 @@
+              <div class="item"><i class="fad fa-check secondary icon"></i>Version your documents with Git.</div>
+              <div class="item"><i class="fad fa-check secondary icon"></i>Test your documentation builds correctly.</div>
+              <div class="item"><i class="fad fa-check secondary icon"></i>Preview the changes before publishing.</div>
+              <div class="item"><i class="fad fa-check secondary icon"></i>Automate the publication process.</div>
+            </div>
+
+            <p>
+              Read the Docs is the <em>"all in one"</em> solution that helps you to make this process easy providing
+              integrated features with zero effort.
+            </p>
+
+          </div>
+        </div>
+
+        <div class="six wide computer twelve wide tablet sixteen wide mobile column">
+          <div class="ui padded basic segment">
+            <div class="ui relaxed list">
+
+              {%- set header_1 = "Traffic and search analytics" %}
+              {%- set icon_1 = "fa-chart-mixed" -%}
+              <a href="#analytics" class="ui small teal header item">
+                <i class="fad {{ icon_1 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_1 }}
+                  <span class="sub header">Know more about how your readers use your documentation.</span>
+                </span>
+              </a>
+
+              {%- set header_2 = "Notifications" %}
+              {%- set icon_2 = "fa-messages" -%}
+              <a href="#notifications" class="ui small header item">
+                <i class="fad {{ icon_2 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_2 }}
+                  <span class="sub header">Warn readers about reading and old version of the documentation.</span>
+                </span>
+              </a>
+
+              {%- set header_3 = "Search as you type" %}
+              {%- set icon_3 = "fa-magnifying-glass" -%}
+              <a href="#search" class="ui small header item">
+                <i class="fad {{ icon_3 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_3 }}
+                  <span class="sub header">Quickly find exactly what you are looking for.</span>
+                </span>
+              </a>
+
+              {%- set header_4 = "Visual diff on pull requests" %}
+              {%- set icon_4 = "fa-file-plus-minus" -%}
+              <a href="#visual-diff" class="ui small header item">
+                <i class="fad {{ icon_4 }} secondary big icon"></i>
+                <span class="content">
+                  {{ header_4 }}
+                  <span class="sub header">Review PR changes visually.</span>
+                </span>
+              </a>
+
+              {%- set header_5 = "Read the Docs authentic flyout" %}
```

Comment:
Not sure what an authentic flyout is? 

```suggestion
              {%- set header_5 = "Read the Docs flyout" %}
```

Reply:
Now that you mentioned it, maybe it doesn't make too much sense.

I thought the flyout as something that people already knew about it, but that's a mistake. In fact, I'm not even sure that people will know what a flyout is at all 😅

Maybe it's better to call it in a more generic way like "version selector" or similar.

Reply:
Well, the second sentence explains what it is, so that may be enough in this context

Reply:
Well, the second sentence explains what it is, so that may be enough in this context.

Reply:
I opened https://github.com/readthedocs/website/pull/233